get_planning_scene_server:
  ros__parameters:
    use_sim_time: true
    point_cloud_topic: "/camera_head/depth/color/points"
    rgb_image_topic: "/camera_head/color/image_raw"
    target_frame: "base_link"
